domain/src/main/java/org/fao/geonet/domain/page/Page.java

```diff
@@ -74,10 +75,11 @@ public Page(PageIdentity pageIdentity, byte[] data, String link, PageFormat form
         this.status = status;
         this.label = label;
         this.icon = icon;
+        this.accessExpression = accessExpression;
     }
 
     public enum PageStatus {
-        PUBLIC, PUBLIC_ONLY, PRIVATE, HIDDEN;
+        PUBLIC, PUBLIC_ONLY, PRIVATE_GROUP, PRIVATE, HIDDEN;
```

Comment:
All groups are private.
I would recommend renaming this to `GROUP`

Reply:
changed